pagopa/pn-frontend
# Pn 8027 fe rifattorizzare il componente empty state

## Short description
Refactored EmptyState component and its usage.

## List of changes proposed in this pull request
- Removed most of the props about empty message structure with children prop in EmptyState component. Now you can use Trans component to make linear and complex empty messages.
- Reworked all the components which use EmptyState
- Reworked translations
- Reworked tests

## How to test
There are no visual changes because it should be the same. However you can test the purpose of EmptyState
Where to test:
- Notification list PA, PF, PG. Add a filter which makes not result.
- Delegates and delegators PA, PF, PG. Remove delegates / delegators to make empty list.
- API Keys in PA. Remove all the API keys to make empty list.
- App Status PA, PF, PG. Make sure to not have any disservice.
